apiserver/facades/client/client/status.go

```diff
@@ -382,6 +382,13 @@ func (c *Client) modelStatus() (params.ModelStatusInfo, error) {
 	return info, nil
 }
 
+type AllApplicationsUnitsBindingsCharmStatus struct {
+	applications     map[string]*state.Application
```

Comment:
should copy existing comment // applications: application name -> application
maybe add comments for other fields?